x/src/lint.rs

```diff
@@ -0,0 +1,164 @@
+use crate::{config::Config, utils::project_root};
+use anyhow::anyhow;
```

Comment:
would really love to see standardized error handling in our code :O

Reply:
I would too, the current crate we're using, failure, is not what we should be using long term.